### ISSUE TITLE
Losing connection to Postgres after 5 minutes idle

### ISSUE THREAD

**Issue by Evadon-Nathan** [ether/etherpad-lite#5040]
I am getting "Error: Connection terminated unexpectedly" and the Etherpad docker container stops if I don't do anything with a pad for about 5 minutes. If I type or refresh the page then it delays this error by another 5 minutes.

I am running Postgres 11.9 from a docker container and Etherpad 1.8.13 from another docker container locally.

I have another service also connecting to the same Postgres instance without issues (although I don't think that one tries to hold connections open so it is unlikely to encounter a disconnect problem like this).

**To Reproduce**
Run Etherpad docker container. Wait 5 minutes.

 - Etherpad version: 1.8.13
 - OS: MacOS
 - Node.js version: 12.8.0
 - npm version: 6.14.8
 - Docker version: 20.10.5

Here is the full error:
```
[2021-05-19 14:37:03.519] [ERROR] server - Metrics at time of fatal error:
{
  "httpStartTime": 1621434685641,
  "memoryUsage": 102727680,
  "memoryUsageHeap": 28835528,
  "ueberdb_lockAwaits": 2,
  "ueberdb_lockAcquires": 187,
  "ueberdb_lockReleases": 187,
  "ueberdb_reads": 133,
  "ueberdb_readsFailed": 0,
  "ueberdb_readsFinished": 133,
  "ueberdb_readsFromCache": 127,
  "ueberdb_readsFromDb": 6,
  "ueberdb_readsFromDbFailed": 0,
  "ueberdb_readsFromDbFinished": 6,
  "ueberdb_writes": 80,
  "ueberdb_writesFailed": 0,
  "ueberdb_writesFinished": 80,
  "ueberdb_writesObsoleted": 2,
  "ueberdb_writesToDb": 78,
  "ueberdb_writesToDbFailed": 0,
  "ueberdb_writesToDbFinished": 78,
  "ueberdb_writesToDbRetried": 0,
  "totalUsers": 1,
  "activePads": 1,
  "httpRequests": {
    "meter": {
      "mean": 0.0059398863642894,
      "count": 2,
      "currentRate": 0.0059398838345531115,
      "1MinuteRate": 0.00015655816029291555,
      "5MinuteRate": 0.0022766089009077046,
      "15MinuteRate": 0.0015530758377158666
    },
    "histogram": {
      "min": 6.414699997752905,
      "max": 35.78700000047684,
      "sum": 42.20169999822974,
      "variance": 431.36600372500817,
      "mean": 21.10084999911487,
      "stddev": 20.769352510971743,
      "count": 2,
      "median": 21.10084999911487,
      "p75": 35.78700000047684,
      "p95": 35.78700000047684,
      "p99": 35.78700000047684,
      "p999": 35.78700000047684
    }
  },
  "connects": {
    "mean": 0.006001683699741572,
    "count": 2,
    "currentRate": 0.006001683824011313,
    "1MinuteRate": 0.00017016369840796357,
    "5MinuteRate": 0.002314870342256662,
    "15MinuteRate": 0.0015617280484794588
  },
  "pendingEdits": 0,
  "edits": {
    "meter": {
      "mean": 0.05490125214191653,
      "count": 18,
      "currentRate": 0.054901252560548355,
      "1MinuteRate": 0.0014433525843542115,
      "5MinuteRate": 0.02063106311052912,
      "15MinuteRate": 0.01401142647073039
    },
    "histogram": {
      "min": 105.53330000117421,
      "max": 117.0730000026524,
      "sum": 1996.2019000016153,
      "variance": 14.13344568998289,
      "mean": 110.9001055556453,
      "stddev": 3.7594475245683228,
      "count": 18,
      "median": 111.67090000212193,
      "p75": 114.22112499922514,
      "p95": 117.0730000026524,
      "p99": 117.0730000026524,
      "p999": 117.0730000026524
    }
  },
  "lastDisconnect": 1621435023519,
  "disconnects": {
    "mean": 0.0031756173165623704,
    "count": 1,
    "currentRate": 0.003175617292359465,
    "1MinuteRate": 0.00009913230091391842,
    "5MinuteRate": 0.0011960020605241296,
    "15MinuteRate": 0.000789539934145336
  }
}
[2021-05-19 14:37:03.521] [ERROR] server - Error: Connection terminated unexpectedly
    at Connection.<anonymous> (/opt/etherpad-lite/src/node_modules/pg/lib/client.js:132:73)
    at Object.onceWrapper (events.js:421:28)
    at Connection.emit (events.js:315:20)
    at Connection.EventEmitter.emit (domain.js:467:12)
    at Socket.<anonymous> (/opt/etherpad-lite/src/node_modules/pg/lib/connection.js:107:12)
    at Socket.emit (events.js:327:22)
    at Socket.EventEmitter.emit (domain.js:467:12)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by webzwo0i** [comment]
Thanks for the report! Could you test if it also happens with dbtype "postgrespool" instead of postgres, please?

**Comment by Evadon-Nathan** [comment]
I think that may have fixed it! Thank you so much.

Just out of interest, is that documented anywhere? I've been searching around for hours to find a solution to this.

**Comment by rhansen** [comment]
Fixed in PR #5163 .
